src/JuAFEM.jl

```diff
@@ -14,8 +14,10 @@ import WriteVTK: vtk_grid
 
 export start_assemble, assemble!, end_assemble
 
-export FECellValues, reinit!, shape_value, shape_gradient, shape_divergence, getdetJdV, getquadrule, getfunctionspace, getgeometricspace,
-                     function_value, function_gradient, function_symmetric_gradient, function_divergence, spatial_coordinate
+export FECellValues, FEVectorCellValues
+
+export reinit!, shape_value, shape_gradient, shape_divergence, getdetJdV, getquadrule, getfunctionspace, getgeometricspace,
```

Comment:
And it should only be defined for `FEVectorCellValues`, since `symmetric()` only works for second order tensors.